packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -288,19 +295,45 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
     );
   }
 
+  // Adjusts the margin percentage, based on the width of the svg (inversly propotional)
+  private _adjustBarSpacing(): void {
+    const svgWidth = this.barChartSvgRef.current?.getBoundingClientRect().width;
+    if (svgWidth) {
+      const BAR_SPACING_FACTOR = 200;
```

Comment:
What does BAR_SPACING_FACTOR represent

Reply:
It's a constant value that's directly proportional to the spacing b/w the bars (by increasing it the spacing increases and v.v)

I think it's not very clear I'll try to reword it and convert it to percentages to make it more clear what this represents.